typings/SavedData.d.ts

```diff
@@ -2,7 +2,16 @@ export class SavedData {
   static disableWebStorage(): void;
   static read(name: string): any
   static remove(name: string): void;
-  static write(name: sttring, value: any, tempOnly: boolean): void;
+  static write(name: string, value: any, tempOnly: boolean): void;
+
+  IDBOpen(dbName: string, dbVersion: INT | null, additions: JSON | null, deletions: JSON | null, callback: function | null): void;
```

Comment:
So instead of INT you should be just using `number` and instead of `JSON` you can either use an interface for the object structure (preferable) or just `object`